schedule/yast/raid/raid1_sle_aarch64_lvm.yaml

```diff
@@ -0,0 +1,36 @@
+---
```

Comment:
name of the file should be: `lvm_raid1_aarch64.yaml` but you can delete this file using `bootloader_start` in the one for x86_64 as mentioned above.